qiita_db/data.py

```diff
@@ -1167,6 +1167,22 @@ def data_type(self, ret_id=False):
         return data_type[0]
 
     @property
+    def samples(self):
+        """Returns samples available in this processed data
+
+        Returns
+        -------
+        set of str
```

Comment:
This needs to be updated -- no longer a set. Also use the "generator" keyword (or say that it yields sample IDs or something to tip off the reader that this is not constructing the set and returning it)